### ISSUE TITLE
Demo build?

### ISSUE THREAD

**Issue by brianxautumn** [brion/ogv.js#452]
Any way you could make a demo build release? Would like to use demo as a submodule or npm package, but there doesn't seem to be a way to do this without cloning the whole repo/submodules and building the entire thing.

**Comment by brion** [comment]
Rough plan:
* separate the demo source to its own repo
* add setup/callback hook for setup and picker options (so can load other JS, set custom options, etc)
* publish that to its own npm module
* you'd then import the 'ogv-demo' module over on 'jsvpx' and create your own webpack config referencing it and building a local output

Let me know if that makes sense and I'll work on it this weekend. :D

**Comment by brianxautumn** [comment]
Yes, demo should for sure have its own npm package. I like the idea of having build options on the demo.
